pages/index.tsx

```diff
@@ -11,10 +11,16 @@ interface HomeProps {
 }
 
 const Home: NextPage<HomeProps> = ({ tasks }) => (
-  <Layout title="Home page">
-    <Header />
-    <Listing tasks={tasks} />
-  </Layout>
+  <>
+    <link
+      href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,700,800"
```

Comment:
to jak już powinno się znaleźć w pliku `_app.js` chcesz do każdego komponentu osobno ładować te czcionki? ;)

Reply:
Usunięte - to było wprowadzone jak walczyłem z wprowadzeniem pogrubnionej czcionki dla strony logowania.